src/client/index.ts

```diff
@@ -169,6 +170,20 @@ function getCollectKeyFromCommand(command: string): string | null {
   }
 }
 
+/**
+ * It returns a function that prepends params to the given func.
+ * A sugar function for JavaScript .bind() without the "this" (keyword) binding.
+ *
+ * @param func - A function to prepend params.
+ * @param params - Parameters to prepend to a function.
+ * @returns A function bound with params.
+ */
+// eslint-disable-next-line @typescript-eslint/ban-types -- expected param types
+function prepend(func: Function, ...params: unknown[]): Function {
```

Comment:
@ledhed2222 Since `partial` will only be used in `Client` (@mvadari), I moved it here and renamed it to `prepend` to tailor it more to `Client` class.